src/lib/scss/colors.scss

```diff
@@ -70,5 +70,6 @@ $frenchLilac: #f0e2f6;
 
 :export {
   amethyst: $amethyst;
+  cornflowerBlueTwo:$cornflowerBlueTwo;
```

Comment:
add space after `:`

Reply:
and in general, use prettier and eslint configs that we have

Reply:
done.